README.md

```diff
@@ -36,7 +36,7 @@ For detailed documentation you can visit
 ## Supported Installation Options
 
 * [Docker / Docker Compose](DOCKER.md)
-* [Setup.bash](https://github.com/DefectDojo/django-DefectDojo/blob/master/setup/README.md)
+* [Setup.bash](https://github.com/DefectDojo/django-DefectDojo/blob/master/setup/README.md)(End of Life 31/12/2020)
```

Comment:
I think we can maybe start adding the helm/k8s deployment option with a warning. But maybe this can also attract contributors, and we're a few now working with the stack. WDYT?

Reply:
sort of out of scope for this PR maybe. not sure what the rest wants. I am unable to provide any support on k8s yet.